plugins/env/app/views/samson_env/_deploy_form.html.erb

```diff
@@ -3,6 +3,13 @@
     Extra Environment Variables for this deploy <%= additional_info SamsonEnv::HELP_TEXT %>
   </legend>
 
+  <div class="form-group">
+    <div class="col-lg-3">
+      <%= form.check_box :use_config_service, id: 'use_config_service',  checked: @deploy.stage.use_config_service %>
```

Comment:
gets an automatic id that we can use

Reply:
should not be settable per deploy or this gets us in trouble with auditing I think